regolith/tools.py

```diff
@@ -425,6 +425,78 @@ def filter_grants(input_grants, names, pi=True, reverse=True, multi_pi=False):
     return grants, total_amount, subaward_amount
 
 
+def filter_grants(input_grants, names, pi=True, reverse=True):
```

Comment:
what is this doing here?  I don't think it should be being added

Reply:
I got confused while restoring the additions from the previous PR.